packages/editor/src/components/post-schedule/panel.js

```diff
@@ -12,9 +13,27 @@ import PostScheduleCheck from './check';
 import PostScheduleForm from './index';
 import { usePostScheduleLabel } from './label';
 import PostPanelRow from '../post-panel-row';
+import { store as editorStore } from '../../store';
+import {
+	TEMPLATE_POST_TYPE,
+	TEMPLATE_PART_POST_TYPE,
+	PATTERN_POST_TYPE,
+	NAVIGATION_POST_TYPE,
+} from '../../store/constants';
+
+const DESIGN_POST_TYPES = [
```

Comment:
We're starting to use this pattern in lots of places and it would be good to handle this more gracefully (maybe a selector). Let's see to it separately though.

Reply:
Maybe a post-type setting indicating that it's intended for the design? It doesn't need to be a public API, just like - `_builtin`.